cordapp-upgrades/contracts/v1-contract/build.gradle

```diff
@@ -0,0 +1,75 @@
+apply plugin: 'kotlin'
+apply plugin: 'net.corda.plugins.cordapp'
+apply plugin: 'net.corda.plugins.cordformation'
+apply plugin: 'net.corda.plugins.quasar-utils'
+
+jar {
+    baseName = "obligation"
+    archiveName = "obligation-contracts.jar"
+    exclude '**/log4j2*.xml'
```

Comment:
Why are we having to do this? Are we including  log4j as a compile dependency?

Reply:
This is a bit of a hangover from the original app I think - I'll look into what's going on here.